modules/WebsiteCMS/WebsiteThemeSetting/Database/Seeders/DefaultWebsiteThemeSettingSeeder.php

```
@@ -0,0 +1,101 @@
+<?php
+
+namespace Modules\WebsiteCMS\WebsiteThemeSetting\Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Modules\Shared\Media\Services\FileUploadService;
+use Modules\WebsiteCMS\WebsiteThemeSetting\Models\WebsiteThemeSetting;
+use Modules\WebsiteCMS\WebsiteThemeSetting\Models\WebsiteThemeSettingDepartment;
+
+class DefaultWebsiteThemeSettingSeeder extends Seeder
+{
+    public function __construct(private FileUploadService $fileUploadService)
+    {
+    }
+
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        // Create default theme setting
+        $defaultTheme = WebsiteThemeSetting::firstOrCreate(
+            ['is_default' => true],
+            [
+                'title' => [
+                    'ar' => 'الثيم الافتراضي',
+                    'en' => 'Default Theme',
+                ],
+                'description' => [
+                    'ar' => 'هذا هو الثيم الافتراضي للموقع',
+                    'en' => 'This is the default theme for the website',
+                ],
+                'about' => [
+                    'ar' => 'معلومات عن الثيم الافتراضي',
+                    'en' => 'Information about the default theme',
+                ],
+                'is_default' => true,
+                'status' => 1,
+            ]
+        );
+
+        //upload default theme image
+
+        $path = resource_path() . "/images/default-theme.png";
+        try {
+            $file = new \Illuminate\Http\UploadedFile(
+                $path,
+                'new-vision-logo.png',
+                null,
+                null,
+                true
+            );
+
+            $this->fileUploadService->uploadFile($defaultTheme, $file, 'website-theme-setting/main-image', "main_image", 'public');
+
+        } catch (\Exception $exception) {
```

Empty catch block is not recommended. At minimum, log the exception or handle it appropriately.